app/main/plugins/plugins.js

```diff
@@ -12,6 +12,14 @@ const { MAX_RESULTS, IS_DEV } = require('../../constants');
 const { app } = electron;
 const { PLUGIN_PATH } = utils.paths;
 
+const resolveByCondition = (condition, Right, Left) =>
+  condition ? Right() : Left();
+const resolvePath = (directoryPath, file) => path.resolve(directoryPath, file);
```

Comment:
everything can be a module in my world.
Using core/global functions in node/browser makes mocking really hard and tedious..

Wrapping them into a util, we isolate the implementation, and the util itself can be mocked.